### PR TITLE
[DOC] Setup

### PR DESCRIPTION
**Pull request by nicolasreymond**
Add Set up section into README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+
+# Setup
+
+1. Communicate with [@BotFather](https://t.me/BotFather) to create a bot
+and get the token.
+1. Rename `secrets.sample.json` to `secrets.json`.
+1. Replace the `BOT_TOKEN` key in the `secrets.json` file with the
+   bot token.
+1. Use `npm i` to create node module file.
+1. Lancer le script `node index.js`.
```
Comment (ponsfrilus) [inline README.md]: WTH this line is in french ?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+[Telegram](https://t.me) bot to manage the consumption of [Club Mate](https://fr.wikipedia.org/wiki/Club-Mate) between different people.
+
+# Setup
+
+1. Communicate with [@BotFather](https://t.me/BotFather) to create a bot
+and get the token.
+1. Rename `secrets.sample.json` to `secrets.json`.
+1. Replace the `BOT_TOKEN` key in the `secrets.json` file with the
+   bot token.
+1. Use `npm i` to create node module file.
```
Comment (ponsfrilus) [inline README.md]: I would suggest: 
 1. Run `npm i` to install the node dependencies.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,24 @@
 # Mate Bot
 
-[Telegram](https://t.me) bot to manage the consumption of
-[Club Mate](https://fr.wikipedia.org/wiki/Club-Mate) between different people.
+[Telegram](https://t.me) bot to manage the consumption of [Club Mate](https://fr.wikipedia.org/wiki/Club-Mate) between different people.
+
+# Setup
+
+1. Communicate with [@BotFather](https://t.me/BotFather) to create a bot
+and get the token.
```
Comment (ponsfrilus) [inline README.md]: Would be nicer to align the second line, as in line 11.

**Review by JaavLex — APPROVED** [review]
Seems good!

**Pull request merged**